.github/workflows/Android.yml

```
@@ -0,0 +1,110 @@
+name: Android
+on:
+  workflow_call:
+    inputs:
+      override_git_describe:
+        type: string
+      git_ref:
+        type: string
+      skip_tests:
+        type: string
+  workflow_dispatch:
+    inputs:
+      override_git_describe:
+        type: string
+      git_ref:
+        type: string
+      skip_tests:
+        type: string
+  repository_dispatch:
+  push:
+    branches:
+      - '**'
+      - '!main'
+      - '!feature'
+    paths-ignore:
+      - '**.md'
+      - 'tools/**'
+      - '!tools/odbc/**'
+      - '!tools/shell/**'
+      - '.github/patches/duckdb-wasm/**'
+      - '.github/workflows/**'
+      - '!.github/workflows/Android.yml'
+
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+    paths-ignore:
+      - '**.md'
+      - 'tools/**'
+      - '!tools/odbc/**'
+      - '!tools/shell/**'
+      - '.github/patches/duckdb-wasm/**'
+      - '.github/workflows/**'
+      - '!.github/workflows/Android.yml'
+
+
+concurrency:
+  group: android-${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref || '' }}-${{ github.base_ref || '' }}-${{ github.ref != 'refs/heads/main' || github.sha }}-${{ inputs.override_git_describe }}
+  cancel-in-progress: true
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  OVERRIDE_GIT_DESCRIBE: ${{ inputs.override_git_describe }}
+
+jobs:
+  android:
+    name: Android
+    runs-on: ubuntu-latest
+    container: ubuntu:18.04
+
+    strategy:
+      matrix:
+        arch: [ armeabi-v7a, arm64-v8a ]
+
+    env:
+      ENABLE_EXTENSION_AUTOLOADING: 1
+      ENABLE_EXTENSION_AUTOINSTALL: 1
+      GEN: ninja
+      EXTENSION_STATIC_BUILD: 1
+      DUCKDB_PLATFORM: android_${{ matrix.arch}}
```

I think we might want also:
```suggestion
      DUCKDB_PLATFORM: android_${{ matrix.arch}}
      DUCKDB_CUSTOM_PLATFORM: android_${{ matrix.arch}}
```

this should make so that on extension install the DUCKDB_CUSTOM_PLATFORM is choosen.

This likely should cleaned-up at the Make / CMake, but for now that's the way. DUCKDB_PLATFORM is possibly not necessary, see Pyodide.yml (https://github.com/duckdb/duckdb/blob/main/.github/workflows/Pyodide.yml) that does without, but I am not completely sure, so may also just stay.

---

thanks!